src/components/Navbar/Navbar.scss

```diff
@@ -0,0 +1,38 @@
+.Navbar {
+  width: 100%;
+  height: 50px;
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  top: 0;
+  font-size: 16px;
+  background-color: var(--color-white);
+  
+  &__left {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+  }
+  
+  &__right {
+    flex: 3;
+    gap: 15px;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    
+    .navbar__links {
+      font-family: var(--inter);
```

Comment:
pls add  text-decoration: none; to remove underline 
maybe also add font properties as per design (line-height, font-size: 14px here)